conans/client/generators/cmake_find_package_multi.py

```diff
@@ -44,6 +46,15 @@ class CMakeFindPackageMultiGenerator(CMakeFindPackageGenerator):
         endforeach()
         """)
 
+    alias_template = Template(textwrap.dedent("""\
+        {%- for alias, target in aliases.items() %}
+        if(NOT TARGET {alias})
+            add_library({{ alias }} INTERFACE IMPORTED)
```

Comment:
Why not ``add_library(<name> ALIAS <target>)``?

Reply:
I think we're using it due to some compatibility issues we had with `ALIAS`?
```
New in version 3.11: An ALIAS can target a GLOBAL Imported Target

New in version 3.18: An ALIAS can target a non-GLOBAL Imported Target. Such alias is scoped to the directory in which it is created and below. The ALIAS_GLOBAL target property can be used to check if the alias is global or not.
```
so it doesn't require very latest cmake to work properly for all the cases